.github/workflows/test-windows.yml

```
@@ -31,6 +31,11 @@ jobs:
     steps:
     - uses: actions/checkout@v1
 
+    - uses: actions/checkout@v1
+      with:
+        repository: python-pillow/pillow-depends
+        ref: master
```

```suggestion
        ref: master
        fetch-depth: 1
```

The repo isn't particularly deep, but we might be able to save some time (from the current ~16s) by only fetching the last commit. Docs:

* https://github.com/actions/checkout/blob/50fbc622fc4ef5163becd7fab6573eac35f8462e/action.yml#L17-L18

---

It seems to be slower, running for about ~18s: https://github.com/nulano/Pillow/runs/315950425